renovate.json

```diff
@@ -1,22 +1,22 @@
 {
     "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-    "labels": [
-        "dependencies"
-    ],
-    "extends": [
-        "config:recommended"
-    ],
-    "ignorePresets": [
-        ":ignoreModulesAndTests"
-    ],
-    "ignorePaths": [
-        "Build/**"
-    ],
-    "ignoreDeps": [
-        "system.threading.tasks.extensions",
-        "Microsoft.NET.Test.Sdk"
-    ],
+
+    // https://docs.renovatebot.com/presets-config/#configrecommended
+    "extends": ["config:recommended"],
```

Comment:
at day work I inherit best-practices, which adds some useful config in addition to the recommended config

This lists the difference: https://docs.renovatebot.com/upgrade-best-practices/ but I think there is a new behavior as well: the minimum release date >3 days (useful for at least npm, where packages can be unpublished three days after push to npmjs.org and for giving a little more room for security researchers to find malicious packages.. Google "npm supply-chain attacks in latest past" or similar